.github/dependabot.yml

```diff
@@ -45,6 +47,9 @@ updates:
       - dependency-name: cloud.google.com/go/*
     groups:
       # Cloud providers' SDK dependencies
+      aws-sdks:
+        patterns:
+          - "github.com/aws/aws-sdk-go-v2/*"
```

Comment:
Yep, it makes sense! I created separate PRs to facilitate this kind of conversation.

Do we want to group these libraries in the AWS SDK (also adding to `groups.aws-sdks.patterns`), or is it better to keep them separate (by only including them in the `allows` list)?

We grouped all the AWS SDK in a group with the `github.com/aws/aws-sdk-go-v2/*` pattern to avoid dependencies problems like https://github.com/elastic/beats/pull/39454 (a PR accidentally upgraded only the core library, breaking main).

Reply:
I suggest adding github.com/aws/aws-lambda-go and github.com/aws/smithy-go as dependencies to check. If we are unsure now, we can still add them later to the group.